README.md

```diff
@@ -1,58 +1,108 @@
+* [Quay](https://quay.io/repository/alfresco/alfresco-base-java) image name: `quay.io/alfresco/alfresco-base-java`
+
+Example final image: `alfresco/alfresco-base-java:jre11-centos7`
+
+> If you are using this base image in a public repository, please stick to the DockerHub published image.
+
+### Image pinning
+
+These tags get overwritten to always have an up-to-date image and hopefully
+without security issues.
+
+If you want to control the updating process of the image, you can use the digest
+in addition to the tag in your `Dockerfile`, for example:
+
+```dockerfile
+FROM alfresco/alfresco-base-java:jre11-centos7@sha256:59a453e01fd958a3748a2e9b0ca99cdf3410f98eeb245499c7bb31696e35bdf4
+```
+
+To discover the latest image tag, just run a docker pull and copy the `Digest`.
 
-Where JAVA_VERSION could be many different things (major version, full version, fullversion with digest...)
+```sh
+docker pull quay.io/alfresco/alfresco-base-java:jre11-centos7
+# jre11-centos7: Pulling from alfresco/alfresco-base-java
+# ...
+# Digest: sha256:59a453e01fd958a3748a2e9b0ca99cdf3410f98eeb245499c7bb31696e35bdf4
+# Status: Downloaded newer image for quay.io/alfresco/alfresco-base-java:jre11-centos7
+```
+
+This configuration is compatible with [Dependabot](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#configuration-options-for-private-registries).
+
+## Development
+
+### Naming specs
+
+The images built from this repository are named as follow:
 
-> Previous tags remain available but we recommend using the new ones
+`<JAVA_DISTRIBUTION_TYPE><JAVA_MAJOR_VERSION>-<OS_DISTRIBUTION_NAME><OS_DISTRIBUTION_VERSION>`
+
+Previous versions of this repository built images using the naming convention:
 
-## Introduction
+`<JAVA_VERSION>[-centos-7]`
 
-This repository contains the [Dockerfile](Dockerfile) used to create the base Java image that will be used by Alfresco engineering teams,
-other internal groups in the organisation, customers and partners to create Java images from.
+Where JAVA_VERSION could be many different things (major version, full version, full version with digest...)
 
-## Pre-requisites
+> Previous tags are still available but are not getting updates anymore
 
-While any docker CLI compatible installation will produce valid  images, [Docker buildx](https://docs.docker.com/buildx/working-with-buildx/) has proven being more efficient and clever when building images using [Multistage builds](https://docs.docker.com/develop/develop-images/multistage-build/). We recommend using it.
+### Prerequisites
 
-## Versioning
+While any docker CLI compatible installation will produce valid images,
+[Docker buildx](https://docs.docker.com/buildx/working-with-buildx/) has proven being
+more efficient and clever when building images using
+[Multistage builds](https://docs.docker.com/develop/develop-images/multistage-build/).
+We recommend using it.
```

Comment:
I know that comes from my own doc but that plain worng. it's not buildx but buildkit: (https://docs.docker.com/develop/develop-images/build_enhancements/). Worth updating in some way so people do not bother installing buildx for nothing